oonf-dlep-proxy/files/dlep_proxy.init

```diff
@@ -3,4 +3,6 @@
 START=82
 DAEMON='dlep_proxy'
 
-. /lib/functions/oonf_init.sh
+[ -n "$IPKG_INSTROOT" ] || {
```

Comment:
I am not sure if I understand this. This will just silence an error?

Reply:
IPKG_INSTROOT is an indicator for IB is running. Don't do anything run time specific.
e.g.
https://github.com/openwrt/openwrt/blob/7b07c3cff57f057d6780d34adeb23c06123732db/package/base-files/files/lib/functions.sh#L52

Reply:
What does IB stand for?

Reply:
openwrt target imagebuilder

Reply:
got it, thank you